client/views/list/list.js

```diff
@@ -693,5 +693,10 @@ Template.list.events({
   'blur .replyarea': function(event, template) {
     Template.instance().state.set('typing', false);
   },
+  "click .closecontainer": function(event, template) {
```

Comment:
Let's use `jquery` here and use the single quote.

Reply:
Updated the quotes, as well as jqeury implementation.